app/cronjob/trailingTrade/step/save-data-to-cache.js

```diff
@@ -19,11 +20,18 @@ const execute = async (logger, rawData) => {
     return data;
   }
 
-  cache.hset(
-    'trailing-trade-symbols',
-    `${symbol}-processed-data`,
-    JSON.stringify(data)
-  );
+  const filter = { symbol };
+
+  const document = _.omit(data, [
+    'closedTrades',
+    'accountInfo',
+    'symbolConfiguration.symbols'
+  ]);
+  if (document.tradingView) {
+    document.tradingView = JSON.stringify(document.tradingView);
```

Comment:
@habibalkhabbaz 

I think this process can be updated.

If you see this screenshot, you will see 3 symbols TradingView is not updated.
![image](https://user-images.githubusercontent.com/5715919/179402655-09e58a98-054d-4b94-afa7-5ebfdc5b3a29.png)

However, if you see the cache, it's updated. 
![image](https://user-images.githubusercontent.com/5715919/179402686-c3ceaa44-b1a1-42fe-861b-a3c6fc8c6425.png)

The reason it's not updating in the frontend because there was no ticker received from WebSocket.
It can be happening the coin that is not active. It's ok with active coins. 

However, if you have the non-active coin, then the frontend looks like not updating, which is not true

This issue also impacts when saving symbol settings for non-active coins. 

1. Get TradingView in `latest.js`. Make sure it uses `cache.hgetall` and assigns to symbol data.
2. Symbol setting update/delete should update the cache. 

If you agree with the above solution, I can work on it. Or you can if you have time. :)

Reply:
Sure! I understood it.
This is the same for `Action` time but I think it make sense, am I right? Because there was no update on the action itself.
By the way, let's start fixing the two mentioned points for now.
I will update you when I start working on it so we don’t have duplicate work.

Reply:
I have updated the code and it will now show something like this if the price hasn't changed over a min.


![image](https://user-images.githubusercontent.com/5715919/179512406-8c791bbd-d25e-467d-879b-fde10ab68984.png)